packages/core/src/browser/keys.ts

```diff
@@ -302,7 +302,15 @@ export class KeyCode {
         }
     }
 
-    public static toCode(event: KeyboardEvent): string {
+    // keyIdentifier is used to access this deprecated field:
+    // https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyIdentifier
+    public static toCode(event: KeyboardEvent & { readonly keyIdentifier?: string }): string {
```

Comment:
just asking, why do we need to declare the "extended" type here? especially if that field is deprecated, would there be any other compatible types used on call sites? to access it, we still need to check as it's done at L.323. 
what do I miss?

Reply:
I wanted to get rid of the `any` cast. That's all.